bedrock/mozorg/templates/mozorg/contact/spaces/spaces-landing.html

```diff
@@ -21,11 +25,122 @@ <h2>Mozilla Spaces</h2>
         continue to building a brighter future for the Web.
       </p>
 
+      <div class="mzp-l-content locations-wrapper">
+        <ul class="mzp-l-columns mzp-t-columns-two">
+          <!-- San Francisco -->
+          {% call picto(
+            base_el='li',
+            title='San Francisco',
+            image=resp_img(
+              url='img/contact/osm/spaces/mozilla-san-francisco.png',
+              optional_attributes={
+                'class': 'mzp-c-picto-image',
+                'loading': 'lazy',
+                'width': '150',
+                'height': '50'
```

Comment:
_(I haven't run the PR locally so don't see the exact output of this thus disregard pls if not an issue…)_

The images are square, so setting the W×H to 150 × 50px seems strange.

Also the images are 600×600px IIRC so if they are used in 150px boxes they can be downsized to ~300px to save some kBs.